<launch>

  <!-- <rosparam command="load" file="$(find control)/config/gps_config.yaml" /> -->
  <!-- <node pkg="control" type="bridge" name="localization_bridge" output="screen"> </node> -->
  <rosparam command="load" file="$(find mpc)/scripts/config/mpc_config.yaml" />
  <node pkg="mpc" type="controller" name="mpc_controller" output="screen"> </node>

</launch>
